### ISSUE TITLE
Question about debug log

### ISSUE THREAD

**Issue by ThorAsgardDev** [google/ExoPlayer#5465]
Hello,
I'm using ExoPlayer 2.9.4.
I'm trying to play smooth files.

It works using some smooth files manifests, and doesn't work using other smooth files manifest.
"Doesn't work" means that I just have black screen.

Could you please tell me how to monitor/debug what is wrong ? I would like to know if the manifest has been downloaded correctly, if the manifest parsing is ok, etc...
I have tryed to add a listener using player.addListener but I don't get any info (except that onPlayerStateChanged is called 3 times with playbackState values 1, 2 and 3)

Thanks for help.

**Comment by ThorAsgardDev** [comment]
I have added a TransferListener on the DataSource. The manifest seems to be downloaded correctly, but no fragments are downloaded. Any idea where the problem comes from ? Or how to get info to know what is wrong ?

**Comment by ThorAsgardDev** [comment]
It seems that the problem comes from DRM.
For smooth files that are using DRM all the renderers are marked as NO_UNSUPPORTED_DRM.
I have tried to set the widevine level to 3 instead of 1 but the problem is still here.
These smooth manifest files contain a protectionheader tag.

Is it normal that even using Widevine level 3 there are no renderer available ? I have tested on Samsung Galaxy S6 Android 7 and OnePlus 6T Android 9.
To solve this problem, is it correct to try to remove manually the protectionheader tag from the manifest before giving it to exoplayer (but keeping the drmSessionManager) ?
